lte/gateway/python/integ_tests/s1aptests/test_attach_detach_with_non_nat_dhcp_multi_ue.py

```diff
@@ -0,0 +1,94 @@
+"""
+Copyright 2022 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+"""
+
+import time
+import unittest
+
+import s1ap_types
+from integ_tests.s1aptests import s1ap_wrapper
+from s1ap_utils import MagmadUtil
+from util.traffic_util import TrafficUtil
+
+
+class TestAttachDetachWithNonNatDhcpMultiUe(unittest.TestCase):
+
+    def setUp(self):
+        """Initialize before test case execution"""
+        self.magma_utils = MagmadUtil(None)
+
+        self.magma_utils.enable_dhcp_config()
+        self.trf_util = TrafficUtil()
+        self._s1ap_wrapper = s1ap_wrapper.TestWrapper()
+        self.magma_utils.disable_nat()
+
+    def tearDown(self):
+        """Cleanup after test case execution"""
+        self.magma_utils.disable_dhcp_config()
+        self.magma_utils.enable_nat()
+        self.trf_util.clear_leases()
+        self._s1ap_wrapper.cleanup()
+
+    def test_attach_detach_with_non_nat_dhcp(self):
```

Comment:
Please rename to test_attach_detach_with_non_nat_dhcp_multi_ue - undocumented convention.

Reply:
Done